gflownet/envs/cube.py

```diff
@@ -953,9 +1028,11 @@ def _get_logprobs_backward(
         logprobs_increments_rel = torch.zeros(
             (n_states, self.n_dim), dtype=self.float, device=self.device
         )
-        jacobian_diag = torch.ones(
+        log_jacobian_diag = torch.zeros(
             (n_states, self.n_dim), device=self.device, dtype=self.float
         )
+        # Mask of effective dimensions
+        is_effective_dim = ~mask[:, -self.n_dim :]
```

Comment:
This variable too seems unused.

Reply:
https://github.com/alexhernandezgarcia/gflownet/pull/209/commits/d6303ec7490aa35e74213adfb7346f4b315b39ea